### PR TITLE
Allow Symfony5

### PR DESCRIPTION
**Pull request by benji07**

### CODE REVIEW

File: tests/Fixtures/Integration/Symfony/config/config.yml [hunk 1]
```diff
@@ -7,13 +7,13 @@ framework:
...
     session:
         storage_id: 'session.storage.mock_file'
-    templating: false
     assets: false
 
 twig:
     strict_variables: false
     paths:
         '%kernel.project_dir%/templates': tests
+    exception_controller: null
```
Comment (benji07) [inline tests/Fixtures/Integration/Symfony/config/config.yml]: this break sf3.4

File: composer.json [hunk 2]
```diff
@@ -31,35 +31,35 @@
...
-        "symfony/framework-bundle": "^3.4|^4.2",
-        "symfony/http-foundation": "^3.4|^4.2",
-        "symfony/serializer": "^3.4|^4.2",
-        "symfony/translation": "^3.4|^4.2",
-        "symfony/twig-bundle": "^3.4|^4.2",
-        "symfony/validator": "^3.4|^4.2",
-        "symfony/templating": "^3.4|^4.2",
-        "symfony/yaml": "^3.4|^4.2",
-        "symfony/var-dumper": "^3.4|^4.2"
+        "symfony/phpunit-bridge": "^3.4|^4.2|^5.0",
```
Comment (ogizanagi) [inline composer.json]: Let's use `"^3.4|^4.4|^5.0"`

File: composer.json [hunk 3]
```diff
@@ -31,35 +31,35 @@
     "prefer-stable": true,
     "minimum-stability": "dev",
     "require": {
-        "php": ">=7.0"
+        "php": ">=7.2"
```
Comment (ogizanagi) [inline composer.json]: Should we really bump the min-php version? Did you encountered any issue with it for this PR?

Comment (benji07) [reply composer.json]: i got a problem with the return type in phpunit classes

Comment (benji07) [reply composer.json]: > 1) Elao\Enum\Tests\Integration\Bridge\Doctrine\DBAL\Type\EnumTypeTest::testEnumType
TypeError: Return value of Elao\Enum\Tests\Integration\Bridge\Doctrine\DBAL\Type\EnumTypeTest::setUp() must be an instance of Elao\Enum\Tests\Integration\Bridge\Doctrine\DBAL\Type\void, none returned

**Comment by ogizanagi** [comment]
Well done. Thx @benji07 !

**Pull request merged**